samples/Mvc.DynamicCorsPolicyProvider/project.json

```diff
@@ -0,0 +1,35 @@
+{
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Microsoft.Extensions.FileProviders.Physical": "1.2.0-*",
```

Comment:
Curious - can this use the ASP.NET meta-package? If so, that can simplify the number of references here.